samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -130,40 +111,22 @@ let ListBlobsSegmentedInFlatListing(container:CloudBlobContainer) =
         // When the continuation token is null, the last page has been 
         // returned and execution can exit the loop.
 
-        let rec loop continuationToken (i:int) = 
-            async {
-                let! ct = Async.CancellationToken
-                // This overload allows control of the page size. You can return
-                // all remaining results by passing null for the maxResults 
-                // parameter, or by calling a different overload.
-                let! resultSegment = 
-                    container.ListBlobsSegmentedAsync(
-                        "", true, BlobListingDetails.All, Nullable 10, 
-                        continuationToken, null, null, ct) 
-                    |> Async.AwaitTask
-
-                if (resultSegment.Results |> Seq.length > 0) then
-                    printfn "Page %d:" i
-
-                for blobItem in resultSegment.Results do
-                    printfn "\t%O" blobItem.StorageUri.PrimaryUri
-
-                printfn ""
-
-                // Get the continuation token.
-                let continuationToken = resultSegment.ContinuationToken
-                if (continuationToken <> null) then
-                    do! loop continuationToken (i+1)
-            }
-    
-        do! loop null 1
+        let blobs = new List<BlobItem>()
```

Comment:
```suggestion
        let blobs = List<BlobItem>()
```

I believe it is idiomatic to omit the `new` here since the list is not disposable.